File: Python/Seção06/Exe08.py
#entradas
num = int(input("Insira um número: "))
#processamento
if num > 0:
    if num % 2 == 0:
        print("{0} é Positivo e Par".format(num))
    else:
        print("{0} é Positivo e Impar".format(num))
else:
    if num % 2 == 0:
        print("{0} é Negativo e Par".format(num))
    else:
        print("{0} é Negativo e Impar".format(num))

File: Java/src/br/com/dominio/s06/Exe08.java
package br.com.dominio.s06;

import java.util.Scanner;

public class Exe08 {

	public static void main(String[] args) {
		//variaveis
		int n;
		Scanner teclado = new Scanner(System.in);

		//entradas
		System.out.println("Insira um número: ");
		n = teclado.nextInt();
		
		//processamento
		if(n % 2 == 0) {
			if (n > 0) {
				System.out.printf("O Número %d é Positivo e Par\n", n );
			}else {
				System.out.printf("O Número %d é Negativo e Par\n", n );
				
			}
			
		}
		if(n % 2 != 0) {
			if (n > 0) {
				System.out.printf("O Número %d é Positivo e Impar\n", n );
			}else {
				System.out.printf("O Número %d é Negativo e Impar\n", n );
			}
				
		}
		teclado.close();
	}

}
